src/devices/Display/README.md

```diff
@@ -0,0 +1,50 @@
+# Display
+
+This project contains multipurpose LED display drivers and binding implementations for concrete display configurations.
+
+## Device family
+
+The **HT16K33** is LED display driver that supports multiple LED configurations and I2C communication.
+
+Adafruit sells multiple display backpacks built upon this driver:\
+[Adafruit LED / SEGMENTED category](https://www.adafruit.com/category/103)
+
+**Large4Digit7SegmentDisplay** is a binding that supports the **Adafruit 1.2" 4-Digit 7-Segment Display w/I2C Backpack** that comes in 3 colors:\
+[Adafruit 1.2" 4-Digit 7-Segment Display w/I2C Backpack - Yellow](https://www.adafruit.com/product/1268)\
+[Adafruit 1.2" 4-Digit 7-Segment Display w/I2C Backpack - Green](https://www.adafruit.com/product/1269)\
+[Adafruit 1.2" 4-Digit 7-Segment Display w/I2C Backpack - Red](https://www.adafruit.com/product/1270)
+
+More information on wiring can be found on the respective product pages.
+
+## Usage
+
+```csharp
+// Initialize display (busId = 1 for Raspberry Pi 2 & 3)
+var display = new Large4Digit7SegmentDisplay(I2cDevice.Create(new I2cConnectionSettings(busId: 1, address: Ht16k33.DefaultI2cAddress));
+
+// Set max brightness (automatically turns on display)
+display.Brightness = display.MaxBrightness;
+
+// Write time to the display
+display.Write(DateTime.Now.ToString("H:mm").PadLeft(5));
+
+// Send buffer to the device
+display.Flush();
```

Comment:
curious, considering this device is relatively short would it make sense to always flush after Write and make Flush private?

Reply:
I think having the Flush method exposed is useful if you want to do multiple buffer modifications and only send them in one go, since every flush operation eventually sends the whole buffer over I2C, which could theoretically lead to intermediate states being displayed, but of course this would be so fast in real life that you wouldn't notice.

What if I added an AutoFlush option to the class?

Reply:
AutoFlush default to true sounds good to me (otherwise I'm expecting we will get an issue soon telling us that this binding doesn't work 😄). Samples can use whichever though